modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -308,7 +318,11 @@ The OTLP gRPC exporter exports data using the OpenTelemetry protocol (OTLP).
 ----
 <1> The OTLP gRPC endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
 <2> The client side TLS configuration. Defines paths to TLS certificates.
-<3> Headers are sent for every RPC performed during an established connection.
+<3> Disables client transport security when set to `true`; `false` by default.
+<4> Skips verifying the certificate when set to `true`; `false` by default.
+<5> Specifies the time interval at which the certificate is reloaded. If not set, the certificate is never reloaded. Accepts a string containing valid units of time such as `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
+<6> When set to a non-empty string, overrides the virtual host name of authority such as the authority header field in requests. Use this for testing.
```

Comment:
Can we also add what this field is? That would mean whatever value the user gives, the corresponding result is expected. We would have to specify `non-empty` string.
Also is `authority header` an example? If not, can we have an actual example?

```suggestion
<6> Overrides the virtual host name of authority such as the authority header field in requests. You can use this for testing.
```